### ISSUE TITLE
Expecting Custom or Full Width Property 

### ISSUE THREAD

**Issue by iftekhariasif** [anuraghazra/github-readme-stats#2607]
### Is your feature request related to a problem? Please describe.

Currently, the Stat Card takes a fixed width (50% I guess). 
So, every time I have to use another element for the same row. 

### Describe the solution you'd like

Having a property for width. 

For Example: 
`![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&bg_color=00000000&width=100%)`

**Comment by rickstaa** [comment]
Does `card_width` solve your problem (see https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options). If I remember it correctly it only accepts `px`.

**Comment by iftekhariasif** [comment]
> Does `card_width` solve your problem (see https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options). If I remember it correctly it only accepts `px`.

Setting up width only in pixel will not be responsive I guess. something in % may be a lifesaver.

**Comment by rickstaa** [comment]
> > Does `card_width` solve your problem (see https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options). If I remember it correctly it only accepts `px`.
> 
> Setting up width only in pixel will not be responsive I guess. something in % may be a lifesaver.

I agree that `%` might make the cards more flexible. You are welcome to create a pull request. I will review it when it is ready. Please, be aware that we use https://en.wikipedia.org/wiki/Percent-encoding for the API endpoint.

**Comment by iftekhariasif** [comment]
> 

Got it. Let me check the source & create a PR. 🙏

**Comment by rickstaa** [comment]
> > 
> 
> Got it. Let me check the source & create a PR. 🙏

Amazing. You can check out https://github.com/anuraghazra/github-readme-stats/blob/2534aec62eeed4c04930a444d23e21c64fe89b30/CONTRIBUTING.md#local-development to see how you can test your changes locally. Feel free to comment below if you have any questions.
